samples/core/SqlServer/Plugin/DbFunctionsExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+
+namespace SqlServer.Plugin;
+public static class DbFunctionsExtensions
```

Comment:
The class should be named something like AugmentationDbFunctionsExtensions

Reply:
Fixed